docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -108,7 +108,61 @@ Here is a complete list of options for Gradle tasks:
 | `allWarningsAsErrors` | Report an error if there are any warnings |  | false |
 | `suppressWarnings` | Don't generate warnings |  | false |
 | `verbose` | Enable verbose logging output. Works only when the [Gradle debug log level enabled](https://docs.gradle.org/current/userguide/logging.html) |  | false |
-| `freeCompilerArgs` | A list of additional compiler arguments |  | [] |
+| `freeCompilerArgs` | A list of additional compiler arguments. You can use experimental `-X` arguments here too. See an [example](#example-of-additional-arguments-usage-via-freeCompilerArgs) |  | [] |
+
+> We are going to deprecate the attribute `freeCompilerArgs` in future releases. If you miss some option in the Kotlin Gradle DSL,
+> please, [file an issue](https://youtrack.jetbrains.com/newissue?project=kt).
+>
+> {type="warning"}
```

Comment:
It doesn't seem like a tip to me, rather a note or even a warning

Reply:
I don't understand; it's already the warning:

> {type="warning"}

Reply:
It was an extra ">" sign before "{type="warning"}", so that's why it was interpreted as a tip, not a warning.